lib/paperclip/matchers/validate_attachment_content_type_matcher.rb

```diff
@@ -54,7 +54,7 @@ def description
 
         def accepted_types_and_failures
           if @allowed_types.present?
-            "Accept content types: #{@allowed_types.join(", ")}\n".tap do |message|
+            "Accept content types: #{@allowed_types.join(", ")}\n".dup.tap do |message|
```

Comment:
Line is too long. [87/80]<br>Prefer single-quoted strings inside interpolations.